package.json

```diff
@@ -21,6 +21,7 @@
   "dependencies": {
     "expo": "^27.0.1",
     "react": "16.3.1",
+    "react-devtools": "^3.2.3",
```

Comment:
from the react-devtools readme https://github.com/facebook/react-devtools/blob/master/packages/react-devtools/README.md, it doesn't sound like you need to add it to the project. They suggest installing it globally (that's the `-g` in the npm command). They run it from the command-line... we'll have to see if that works